xtask/src/main.rs

```diff
@@ -0,0 +1,47 @@
+use std::error::Error;
+use structopt::StructOpt;
+
+mod test_unit;
+
+#[derive(Debug, StructOpt)]
+struct Args {
+    #[structopt(subcommand)]
+    cmd: Command,
+}
+
+#[derive(Debug, StructOpt)]
+enum Command {
+    #[structopt(name="test-unit")]
+    /// Run unit tests
+    TestUnit {
+        #[structopt(long, short)]
+        /// Restrict unit tests to a specific pacakge
+        package: Option<String>,
+    },
+    #[structopt(name="test-system")]
+    /// Run system tests
+    TestSystem,
```

Comment:
Do you want to remove this since it’s unused?

Reply:
Yep.